torch/_torch_docs.py

```diff
@@ -2931,6 +2931,25 @@ def merge_dicts(*dicts):
     tensor([False, True, False])
 """)
 
+add_docstr(torch.isreal,
+           r"""
+isreal(input) -> Tensor
+
+Returns a new tensor with boolean elements representing if each element is real-valued or not.
+All real-valued types are considered real. Complex values are considered real when their imaginary part is 0.
+
+Arguments:
+    input (Tensor): A tensor to check
+
+Returns:
+    Tensor: A boolean tensor containing a True at each location of real elements.
```

Comment:
"a boolean tensor with True where :attr:`input` is real-valued and False elsewhere"